README.coverage.md

```
@@ -0,0 +1,99 @@
+# Test Coverage
+
+Test coverage can be computed by fist configuring and building the code for coverage.
+Once done, run both `make` and `make check`. The build process will produce a number of
+_.gcda and _.gcno files that contain the coverage information. Once that is done, use
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)